### PR TITLE
OCPBUGS#30157: Updates procedure for sharing feedback in web console

### PR DESCRIPTION
**Pull request by opayne1**
OCPBUGS#30157: Updates procedure for sharing feedback in web console

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-30157

Link to docs preview:
https://72563--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data#support-get-cluster-id_gathering-cluster-data

QE review:
- [x] QE has approved this change.
QE ack in JIRA

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
@juzhao PTAL. Thanks!

**Comment by mburke5678** [comment]
@opayne1 One question. Otherwise LGTM

File: modules/support-get-cluster-id.adoc [hunk 1]
```diff
@@ -21,13 +21,8 @@ endif::openshift-rosa,openshift-dedicated[]
 .Procedure
 ifdef::openshift-enterprise,openshift-webscale,openshift-dedicated[]
 * To open a support case and have your cluster ID autofilled using the web console:
-.. From the toolbar, navigate to *(?) Help* -> *Open Support Case*.
-.. The *Cluster ID* value is autofilled.
-endif::[]
-ifdef::openshift-origin[]
```
[the remark this answers is not in the document]
Comment (opayne1) [reply modules/support-get-cluster-id.adoc:27]: I think it was an accident. Thanks!

**Pull request merged**